test/fuzz/README.md

```diff
@@ -0,0 +1,42 @@
+# Fuzz Testing
+
+Fuzz testing is:
+
+> An automated software testing technique that involves providing invalid, unexpected, or random data as inputs to a program.
+
+We use coverage guided fuzz testing to automatically discover bugs in `pdf.js`
+
+This `fuzz/` directory contains the configuration and the fuzz tests for `pdf.js`.
+To generate and run fuzz tests, we use [jazzer.js](https://github.com/CodeIntelligenceTesting/jazzer.js/) library.
+
+## Running a fuzzer
+
+This directory contains fuzzers like for example `jpg.fuzz`. You can run it with:
+
+// generate image decoders
+```sh
+$ gulp imag_decoder
```

Comment:
That command doesn't exist, hence trying to follow these instructions as-is won't work.
```suggestion
$ gulp image_decoders
```